integration_tests/tests/solana_test_validator.rs

```diff
@@ -530,7 +529,7 @@ async fn should_get_signatures_for_address() {
         .compare_client(
             |sol| {
                 sol.get_signatures_for_address_with_config(
-                    &Pubkey::default(), // address of the Solana system program
+                    &system_program::id(), // address of the Solana system program
```

Comment:
nit: now it's self explanatory and the comment is IMO redundant.
```suggestion
                    &system_program::id(),
```